modules/dev-tools/assets/js/deprecation/utils.js

```diff
@@ -0,0 +1,114 @@
+import { consoleWarn } from '../utils';
+
+/**
+ * @typedef {Object} Version
+ * @property {number} major1
+ * @property {number} major2
+ * @property {number} minor
+ * @property {string} build
+ */
+
+export const softDeprecated = ( name, version, replacement ) => {
+	if ( elementorDevToolsConfig.isDebug ) {
+		deprecatedMessage( 'soft', name, version, replacement );
+	}
+};
+
+export const hardDeprecated = ( name, version, replacement ) => {
+	deprecatedMessage( 'hard', name, version, replacement );
+};
```

Comment:
Don't export them

Reply:
They have to be available using `Import`, they are exported in file but not an externals.